src/App.js

```diff
@@ -7,13 +7,13 @@ import Timeline from './components/Timeline';
 function App() {
   console.log(timelineData);
 
-  // Customize the code below
   return (
     <div className="App">
       <header className="App-header">
-        <h1 className="App-title">Application title</h1>
+        <h1 className="App-title">Ada Lovelace's social media feed</h1>
```

Comment:
Just noting that you could do:

```suggestion
        <h1 className="App-title">{timelineData.person}'s social media feed</h1>
```